legal-auto.py

```diff
@@ -178,6 +208,108 @@ def _set_review(self, req, state):
                                      by_user=self.review_user, message=self.message)
         self.delete_from_db(req.reqid)
 
+    def _pkl_path(self):
+        CACHE_DIR = os.path.expanduser('~/.cache/osc-plugin-factory')
```

Comment:
nit: use the new project name and perhaps xdg as done elsewhere.

Reply:
yeah - these are changes done in june 2017 (and used in production since then ;)

Reply:
could you please fix this nevertheless?

Reply:
why merge?